pkg/monitor/monitor/util.go

```diff
@@ -139,6 +142,58 @@ func getMonitorConfigMap(tc *v1alpha1.TidbCluster, dc *v1alpha1.DMCluster, monit
 		EnableTLSDMCluster: dc != nil && dc.IsTLSClusterEnabled(),
 	}
 
+	if monitor.Spec.RemoteWrite != nil && len(monitor.Spec.RemoteWrite) > 0 {
+		var remoteWriteConfigs []*config.RemoteWriteConfig
+		for _, remoteWrite := range monitor.Spec.RemoteWrite {
+			url, err := client.ParseHostURL(remoteWrite.URL)
+			if err != nil {
+				continue
```

Comment:
would be better to output a warning log?

Reply:
util file can’t import log class.